istresearch/mailroom
# P4-3000 - Log when channel cant be found for message

# For the Reviewer

When this is complete, you should approve the PR via github.

# For the Reviewee

<Reminder PR Title should be JIRA_NUMBER and Useful Description>

## Summary
Added a log to the message created handler so when a channel cannot be found, it will log out a message. 

#### Release Note
Added additional log when a channel can't be found for a message

#### Breaking Changes
<Description for Techops of how to handle changes, migrations, updates>None.

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

1. Using a docker stack, adjust mailroom to `istresearch/mailroom:ci-4.0.10-dev-P4-3000`. 
2. `docker logs -f mailroom`
3. Create a contact with a URN for a non-existent channel on your system. 
4. Use `Send Message` to send a message to the user
5. When the message processes, you should see a log similar to the following:
```
{"contact_id":"04b2ff4e-b2f0-4a58-b963-f6875e7fcaf7","level":"error","message":"Could not find channel for message","timestamp":"2021-09-22T17:38:48.466Z"}
```
